Src/Camera/CameraData.cs

```diff
@@ -120,6 +124,7 @@ public static void LoadSettings()
                 ws.WriteLine("	LookAt={x='0.0', y='-0.5', z='1.0'} ");
                 ws.WriteLine("	MinTime='4.0' ");
                 ws.WriteLine("	MaxTime='8.0' ");
+                ws.WriteLine("  TransitionTime='1.0'");
```

Comment:
Replace with tabs

Reply:
Also, do we even want to change the default settings?

Reply:
I've got no problem with that, but we might want to put in more error checking if they values are not found in the loaded settings file.

Reply:
I think that would also be helpful! I think overall some settings changes could be in order to make things easier too~